.github/workflows/ci-app.yml

```diff
@@ -0,0 +1,77 @@
+name: Continuous Integration Delta App
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'delta/sourcing/**'
+      - 'delta/rdf/**'
+      - 'delta/sdk/**'
+      - 'delta/sdk-testkit/**'
```

Comment:
if we also include the sdk-testkit we could just have `delta/**`

Reply:
Not yet, since we have a lot of things in `delta/src/**` which we don't care about on the `app` module